resources/contribute-documentation/index.md

```diff
@@ -84,6 +85,20 @@ If you add a new page, you should add it to the appropriate `sidebars.yaml` file
 
 The Markdown file for a page should start with a [frontmatter stanza](#frontmatter-fields).
 
+## Writing Style
+
+In general, follow the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/)'s recommendations for tone, phrasing, and structure. You are allowed to deviate from the Microsoft guide.
+
+Try to split documentation pages into the following categories, based on their contents:
+
+- **Concepts:** Explain the _how_ and _why_ of a topic, feature, or practice. Don't go overboard with specific names, values, or code. Use [diagrams](./creating-diagrams.md) to help with understanding.
+- **References:** Describe all the possible properties, modes, or features of a piece of technology. Provide only small amounts of context, while linking to concepts for the bigger picture. Use tables or bulleted lists, sorted, to help users find what they need.
+- **Tutorials:** Demonstrate how to accomplish a task or use a feature. Provide step-by-step instructions with working code samples if possible. Link to references for more details of the functions used and alternative options. For more recommendations on how to build tutorials, see the [Tutorial Guidelines](./tutorial-guidelines.md).
+
+These types of documents serve people with different roles, or sometimes the same people at different points in their process. For example, software architects or CTOs making technical decisions might read a concept page to decide how to structure their integration or to determine which technology meets their needs; engineers might copy code from tutorials while building, then use the reference to adapt it for their use case.
+
+Some pages, like **Use Cases**, don't fit cleanly into these categories. It is OK to have exceptions where justified.
+
```

Comment:
Suggest adding a note highlighting the importance of cross-linking related pages for SEO and GEO.